### DISCUSSION TITLE
Vercel build fails on corrupted size vs. prev_size

### DISCUSSION THREAD

**Discussion by liornz** [vercel/community#5448] [Help]
### Summary

local build for a next.js project succeeds but when trying to deploy to vercel I get a build error:
corrupted size vs. prev_size
Command "npm run build" exited with SIGABRT
I have no idea how to fix this as locally build passes without any issues...
I saw in some discussion someone mentioned sharp version so I downgraded to sharp version 0.32.6 as mentioned in that thread, but this does not help either...

### Steps to Reproduce

▲ Next.js 14.0.4
--
23:02:56.228 |  
23:02:56.228 | Creating an optimized production build ...
23:02:56.753 | > [PWA] Compile server
23:02:56.754 | > [PWA] Compile server
23:02:56.755 | > [PWA] Compile client (static)
23:02:56.755 | > [PWA] Auto register service worker with: /vercel/path0/node_modules/next-pwa/register.js
23:02:56.755 | > [PWA] Service worker: /vercel/path0/public/sw.js
23:02:56.756 | > [PWA]   url: /sw.js
23:02:56.756 | > [PWA]   scope: /
23:02:59.992 | corrupted size vs. prev_size
23:05:12.766 | Error: Command "npm run build" exited with SIGABRT

**Comment by swarnava** [comment]
Hi, 

are you able to run `npm run build` locally?

**Reply by liornz** [reply]
Yes... I could.
Somehow this issue just resolved itself. Without me changing anything it now passes build and deploy...
